lib/ansible_tower_client/base_models/project.rb

```diff
@@ -19,8 +19,19 @@ def update
     end
 
     def last_update
-      return if related['last_update'].blank?
-      api.project_updates.find(related['last_update'])
+      return @last_update if defined? @last_update
+      return @last_update = nil unless related.raw_hash.key?('last_update')
+      return @last_update = nil if (update_id = related.last_update).blank?
```

Comment:
Same here?

Reply:
Yes, `defined?` everywhere exactly for this purpose.